arrow-core-data/src/main/kotlin/arrow/core/Ior.kt

```diff
@@ -139,7 +139,8 @@ sealed class Ior<out A, out B> : IorOf<A, B> {
     is Both -> fab(leftValue, rightValue)
   }
 
-  fun <C> foldLeft(c: C, f: (C, B) -> C): C = fold({ c }, { f(c, it) }, { _, b -> f(c, b) })
+  inline fun <C> foldLeft(c: C, f: (C, B) -> C): C =
+    fold({ c }, { f(c, it) }, { _, b -> f(c, b) })
 
   fun <C> foldRight(lc: Eval<C>, f: (B, Eval<C>) -> Eval<C>): Eval<C> =
```

Comment:
@JorgeCastilloPrz this is an example, as you see, because f is passed inside Eval.defer { } and it is stored in a data class, so it cannot be inlined.